addons/web/static/src/views/list/list_renderer.js

```diff
@@ -61,14 +62,70 @@ function getElementToFocus(cell, index) {
     return getTabableElements(cell).at(index) || cell;
 }
 
+export class ListRecordRow extends Component {
+    static props = ["*"];
+    static template = "web.ListRecordRowContainer";
+    static LONG_TOUCH_THRESHOLD = 400;
+    setup() {
+        if (!this.props.record.isNew) {
+            useMiddleClick({
+                clickParams: {
+                    record: this.props.record,
+                },
```

Comment:
I'm not sure that this will work with the custom actions, see: https://github.com/odoo/odoo/commit/a066638cd55ec2a3e74a1916c8fb010d9e23a272

Reply:
Okay, let me check for list view actions.
I skipped kanban implementation in this PR as it was uncomplete. I didn't knew there was also this for the list view